packages/aws-cdk-lib/aws-backup/lib/rule.ts

```diff
@@ -41,9 +42,17 @@ export interface BackupPlanRuleProps {
    * A CRON expression specifying when AWS Backup initiates a backup job.
    *
    * @default - no schedule
+   * @deprecated use schedule prop instead
    */
   readonly scheduleExpression?: events.Schedule;
 
+  /**
+   * A CRON expression specifying when AWS Backup initiates a backup job.
+   *
+   * @default - no schedule
```

Comment:
The word "schedule" now has lost all meaning to me and has turned into an abstract garble of letters 🫣